client/src/helpers/APICalls/search.ts

```diff
@@ -0,0 +1,18 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const search = async (place: string, drop: string): Promise<AuthApiData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ place, drop }),
+    credentials: 'include',
+  };
+  return await fetch(`/auth/search`, fetchOptions)
```

Comment:
Note: We will need to use the profile route here to get a list of all the dog sitters, we can tackle the implementation in another ticket 👍🏽